website/integrations: GlitchTip: cleanup (#23141)
goauthentik/authentik: website/integrations/monitoring/glitchtip/index.md

<RedirectURI20265Note />

To support the integration of GlitchTip with authentik, you need to create an application/provider pair in authentik.

### Create an application and provider in authentik

1. Log in to authentik as an administrator and open the authentik Admin interface.
2. Navigate to **Applications** > **Applications** and click **New Application** to open the application wizard.

>     - **Application**: provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings. Note the **slug** value because you will use it when configuring GlitchTip.

Reviewer: ```suggestion
    - **Application**: provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings. Note the **Slug** value because you will use it when configuring GlitchTip.
```

```shell
sudo docker exec -it glitchtip-web-1 ./manage.py createsuperuser
```

2. Go to `https://glitchtip.company/admin/socialaccount/socialapp/` and log in with the superuser.

3. Click **Add Social Application** and enter the following details:
    - **Provider**: `OpenID Connect`
    - **Provider ID**: `authentik`
    - **Provider Name**: `authentik`
    - **Client ID**: `<Client ID from authentik>`
    - **Secret key**: `<Client Secret from authentik>`
    - **Settings**:

        ```json
        {
            "server_url": "https://authentik.company/application/o/<application_slug>/"
        }
        ```

4. Click **Save**.

The **Provider ID** value determines the callback path. If you use a provider ID other than `authentik`, update the redirect URI in the authentik provider to `https://glitchtip.company/accounts/oidc/<provider_id>/login/callback/`.

To add an authentik account to an existing GlitchTip account, log in with the existing username and password, click **Profile**, and then click **Add Account** in the **Social Auth Accounts** section.

## Configuration verification

> To confirm that authentik is properly configured with GlitchTip, log out of GlitchTip and open the GlitchTip login page. Click the authentik login option and confirm that you are redirected to authentik for authentication and then back to GlitchTip.

Reviewer: ```suggestion
To confirm that authentik is properly configured with GlitchTip, log out of GlitchTip and open the GlitchTip login page. Click the authentik login option. You should be redirected to authentik for authentication and then returned to GlitchTip.
```
